dev/io.openliberty.security.jakartasec.3.0.internal/resources/io/openliberty/security/jakartasec/internal/resources/JakartaSecurity30Messages.nlsprops

```diff
@@ -52,3 +52,11 @@ JAKARTASEC_CLAIMS_PROCESSING_ERROR.useraction=For more information, see the erro
 JAKARTASEC_WARNING_OIDC_MECH_CONFIG_NEGATIVE_INT=CWWKS2507W: The value of the {0} attribute of the OpenID authentication mechanism definition annotation must not be a negative integer. If the attribute value is an Expression Language (EL) expression, it must not resolve to a negative integer. The value provided was ''{1}''. The default attribute value of ''{2}'' is used instead.
 JAKARTASEC_WARNING_OIDC_MECH_CONFIG_NEGATIVE_INT.explanation=The value of the attribute must not be a negative integer. If the attribute value is an EL expression, the value must not resolve to a negative integer.
 JAKARTASEC_WARNING_OIDC_MECH_CONFIG_NEGATIVE_INT.useraction=Update the attribute to use a non-negative integer. If the attribute value is an EL expression, update the referenced bean to resolve to a non-negative integer.
+
+CREDENTIAL_VALIDATION_OMISSION=CWWKS2508E: The credential cannot be validated due to missing data. Either the token response ''{0}'', client ''{1}'' or access token ''{2}'' were missing.
+CREDENTIAL_VALIDATION_OMISSION.explanation=Credential validation requires a value for the response token, client or access token. At least one of these values was null.
+CREDENTIAL_VALIDATION_OMISSION.useraction=For more information, see the error in the message.
+
+CREDENTIAL_VALIDATION_CALLER_MISSING=CWWKS2509E: The credential cannot be validated because the caller value is not set.
```

Comment:
```suggestion
CREDENTIAL_VALIDATION_CALLER_MISSING=CWWKS2509E: The credential cannot be validated because the caller value is not set. The [{0}] caller name claim is null or empty, or a claim with that name cannot be found in the access token, the ID token, or the UserInfo data for the user.
```